vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java

```diff
@@ -57,6 +57,7 @@ public void initView() {
         createFormLayoutWithItems();
         createFormLayoutWithBinder();
         createCompositeLayout();
+        createLayoutHandleColspans();
```

Comment:
Actually, I think this demo could be shown already as the third demo after `createFormLayoutWithItems()`. Please move it there.